cloud/pkg/common/messagelayer/util.go

```diff
@@ -62,6 +62,9 @@ func BuildResource(nodeID, namespace, resourceType, resourceID string) (resource
 func getElementByIndex(msg model.Message, index int) string {
 	sli := strings.Split(msg.GetResource(), constants.ResourceSep)
 	if len(sli) <= index {
+		if index == ResourceResourceTypeIndex {
+			return "node"
+		}
```

Comment:
How come we can say this? we are not sure what is missing in the resource path? Do we have issue for this?

Reply:
I am sorry i am not following your explanation. can you rephrase this a bit?

Reply:
I see this ticket for "Object 'Kind' is missing" issue and a possible solution in its comment: https://github.com/kubeedge/kubeedge/issues/2956#issuecomment-1437702572.

So, I was trying to find out which messages are for getting node but without specifying the "Kind" by checking `len(sli)`.

Reply:
I am not really sure the relationship between this specific work-around and https://github.com/kubeedge/kubeedge/issues/2956... could you please elaborate a bit? or you are not sure about this either?

Reply:
> If len(sli) == 3, the request message makes no sense, because when a normal application wants to get node, it does not specify namespace, and it is not needed by API server for providing node information.

i see this does not make sense though.